### ISSUE TITLE
Email generator does not generate proper emails

### ISSUE THREAD

**Issue by magro** [kotest/kotest#1302]
Running the following test fails:
```
"email generator" {
    checkAll(1, Arb.email()) { email ->
        email shouldNot contain(" ")
    }
}
```

Test failure:
```
Property failed after 1 attempts

	Arg 0: "b, e, z, c, a, r, b, c@b, d, t, b, e, e, p, u.[com, net, gov, co.uk, jp, nl, ru, de, com.br, it, pl, io]"
```

Not sure if I'm using that wrong, or if it's a bug.

For reference: email generator was added with commit d8befa16 to solve #1134

**Comment by sksamuel** [comment]
Fixed in master. Will be released as part of 4.0 final.

**Comment by magro** [comment]
Awesome, thanks!
